### PR TITLE
chore: remove TravisCI link

### PR DESCRIPTION
**Pull request by bnb**
Pretty sure that basically nobody in the CPC would recommend Travis after what's happened with them. We probably shouldn't list it here.

### CODE REVIEW

**Review by WaleedAshraf — APPROVED** [review]

File: project-resources/INFRASTRUCTURE_MENU.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ The OpenJS Foundation will purchase 2-year wildcard SSL certs for each project's
 By default all OpenJS Foundation projects have open source repositories in their own GitHub Organizations.  The OpenJS Foundation admin account must be added as administrator for each repository.  Two-factor authentication must be required for everyone in the organization.
 
 ## Build (CI/CD)
-Projects which require CI/CD should attempt to use solutions which are free to open source projects, such as https://travis-ci.org/.  Non-free options may be requested, subject to budget approval.
+Projects which require CI/CD should attempt to use solutions which are free to open source projects.  Non-free options may be requested, subject to budget approval.
```
Comment (ljharb) [inline project-resources/INFRASTRUCTURE_MENU.md:28]: alternative:
```suggestion
Projects which require CI/CD should attempt to use solutions which are free to open source projects, such as Github Actions. Non-free options may be requested, subject to budget approval.
```

**Review by ljharb — APPROVED** [review]

**Review by tobie — APPROVED** [review]

File: project-resources/INFRASTRUCTURE_MENU.md (hunk 1 above)
Comment (tobie) [reply project-resources/INFRASTRUCTURE_MENU.md:28]: I'm OK either way.

Comment (bnb) [reply project-resources/INFRASTRUCTURE_MENU.md:28]: I'd be okay with this but also medium -1 against us including free tools that projects can get themselves in the Foundation's "Infrastructure Menu". Feels wildly underwhelming and useless.

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**